set.mm

```diff
@@ -440751,22 +440862,92 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
       STVMVFWAWBXDVNWEWLWTVOVRTVPVSVT $.
   $}
 
-$(
-  @{
-    locfinref.x @e |- X = U. J @.
-    locfinref.1 @e |- ( ph -> U C_ J ) @.
-    locfinref.2 @e |- ( ph -> X = U. U ) @.
-    locfinref.3 @e |- ( ph -> V C_ J ) @.
-    locfinref.4 @e |- ( ph -> U Ref V ) @.
-    locfinref.5 @e |- ( ph -> V e. ( LocFin ` J ) ) @.
-    @( A locally finite refinement of an open cover induces a locally finite
-       cover with the original index set.  (Contributed by Thierry Arnoux,
-       12-Jan-2020.) @)
-    locfinref @p |- ( ph -> E. w e. ~P J ( U Ref w /\ w e. ( LocFin ` J ) ) )
-      @=
-      ? @.
-  @}
-$)
+  ${
+    $d J f g j k u v w x $.  $d U f g j k u v w x $.  $d V f g j k u v w x $.
+    $d X j k u v w x $.  $d f g j k u v w x ph $.
+    locfinref.x $e |- X = U. J $.
+    locfinref.1 $e |- ( ph -> U C_ J ) $.
+    locfinref.2 $e |- ( ph -> X = U. U ) $.
+    locfinref.3 $e |- ( ph -> V C_ J ) $.
+    locfinref.4 $e |- ( ph -> U Ref V ) $.
+    locfinref.5 $e |- ( ph -> V e. ( LocFin ` J ) ) $.
+    $( A locally finite refinement of an open cover induces a locally finite
+       cover with the original index set.  This is fact 2 of
+       http://at.yorku.ca/p/a/c/a/02.pdf , it is expressed by exposing a
+       function ` f ` from the original cover ` U ` , which is taken as the
+       index set.  (Contributed by Thierry Arnoux, 29-Jan-2020.) $)
+    locfinref $p |- ( ph -> E. f ( ( Fun f /\ dom f C_ U )
+      /\ ( U Ref ran f /\ ran f e. ( LocFin ` J ) ) ) ) $=
```

Comment:
This looks like a strange way to state this theorem. What is the purpose of the function `f`? It seems like all that matters is `ran f`, and the only thing about the refinement one can derive from the fact that `dom f C_ U` is `ran f ~<= U`. So why not say this instead?